assemble/bin/accumulo-cluster

```diff
@@ -100,15 +100,45 @@ function get_ip() {
   echo "$ip_addr"
 }
 
+function set_last_instance_id() {
+  if [[ "$service" == "tserver" ]]; then
+    last_instance_id=${NUM_TSERVERS:-1}
+  else
+    last_instance_id=1
+  fi
+}
+
+function set_service_instance_if_needed() {
+  if [[ "$service" == "tserver" && ${NUM_TSERVERS:-1} > 1 ]]; then
+    export ACCUMULO_SERVICE_INSTANCE=$inst_id
+    service_instance_cmd="export ACCUMULO_SERVICE_INSTANCE=${inst_id};"
+  else
+    export ACCUMULO_SERVICE_INSTANCE=
+    service_instance_cmd=
+  fi
+}
+
+function control_service() {
+  set_last_instance_id
+
+  for (( inst_id=1; inst_id<=last_instance_id; inst_id++ ))
+  do
+    set_service_instance_if_needed
+
+    if [[ $host == localhost || $host = "$(hostname -s)" || $host = "$(hostname -f)" || $host = $(get_ip) ]] ; then
+      "${bin}/accumulo-service" "$service" "$control_cmd"
+    else
+      $SSH "$host" "bash -c '${service_instance_cmd} ${bin}/accumulo-service \"$service\" \"$control_cmd\"'"
```

Comment:
Instead of creating a second variable and exporting here, you can probably just execute it with the environment variable set for the execution:

```suggestion
      $SSH "$host" "bash -c 'ACCUMULO_SERVICE_INSTANCE='"$ACCUMULO_SERVICE_INSTANCE"' ${bin}/accumulo-service \"$service\" \"$control_cmd\"'"
```

Reply:
Ah - I think the usage of the semicolon in my script was the reason the variable was not visible within the accumulo-service script. I guess I was going off the syntax in the [docs](https://accumulo.apache.org/docs/2.x/administration/in-depth-install#running-multiple-tabletservers-on-a-single-node) which has a semicolon between the ACCUMULO_SEVICE_INSTANCE variable and the accumulo script. I wonder now if that is a typo which needs to be addressed?

Reply:
Ah, yeah, that's definitely a typo in the docs. I'll fix that.

Reply:
The docs will further need to be updated to mention the use of `NUM_TSERVERS` for 2.1 and later, once this PR gets merged.

Reply:
Thank you and I did plan to submit a docs PR for `NUM_TSERVERS` usage when this change is done.

Reply:
The changes have been done - please review when you have a chance.